bedrock/firefox/urls.py

```diff
@@ -36,12 +36,6 @@
     url(r'^firefox/desktop/$',
         views.FirefoxProductDesktopView.as_view(),
         name='firefox.desktop.index'),
-    page('firefox/desktop/fast', 'firefox/desktop/fast.html', alt_canonical_paths={
-        'en-US': '/firefox/features/fast/',
-    }),
-    page('firefox/desktop/customize', 'firefox/desktop/customize.html'),
-    page('firefox/desktop/tips', 'firefox/desktop/tips.html'),
-    page('firefox/desktop/trust', 'firefox/desktop/trust.html'),
```

Comment:
There's still a reference to `url('firefox.desktop.trust')` in `firefox/dnt.html`

Reply:
Bah, nice spot. 🕵️